access/src/main/resources/holdingsitems/migration/V5__ThreeLevelStructure.sql

```diff
@@ -62,6 +62,8 @@ ALTER TABLE bibliographicitem
   ALTER COLUMN modified SET DEFAULT timeofday()::timestamp;
 ALTER TABLE bibliographicitem
   ALTER COLUMN trackingid SET NOT NULL;
+ALTER TABLE bibliographicitem
+    ADD COLUMN version INTEGER NOT NULL DEFAULT 0;
```

Comment:
Jeg vil gerne have at ALTER TABLE ... ADD COLUMN bliver ændret til at feltet bliver oprettet i CREATE TABLE ... AS SELECT, og der så kun bliver sat NOT NULL constraint på i ALTER TABLE.
Hvis man ikke gør dette skal hele tabellen traverseres igennem igen for at oprette feltet på hver række.